src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
  * removed `TraceableControllerResolver::getArguments()`
  * removed `ControllerResolver::getArguments()` and the ability to resolve arguments
  * removed the `argument_resolver` service dependency from the `debug.controller_resolver`
+ * removed `Extension::addClassesToCompile()`
```

Comment:
`loadClassCache()` and `setClassCache()` of the `Kernel` class have been removed too